tests/publiccloud/download_repos.pm

```diff
@@ -27,20 +27,19 @@ sub run {
     my ($self, $args) = @_;
     select_host_console();    # select console on the host, not the PC instance
 
-    # Skip maintenance updates. This is useful for debug runs
-    my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
+    assert_script_run("mkdir ~/repos");
```

Comment:
why do we need to create folder when we skipping ?

Reply:
Because other tests e.g. `transfer_repos` depend on the existence of this folder.

Reply:
> Because other tests e.g. `transfer_repos` depend on the existence of this folder.

and it is already correctly handled https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/publiccloud/transfer_repos.pm#L28